src/App.js

```diff
@@ -1,23 +1,9 @@
-import logo from './logo.svg';
-import './App.css';
+import Container from './components/Container';
 
 function App() {
```

Comment:
- [ ] Test cases/ suites are failing. Kindly ensure that all test cases are passing/ green.

Failing test case:
![image](https://user-images.githubusercontent.com/12745474/142739811-983922e3-f744-4dca-8c91-f21d29a84ead.png)